utils/ort/src/main/kotlin/Utils.kt

```diff
@@ -37,7 +37,8 @@ var printStackTrace = false
 private val versionSeparators = listOf('-', '_', '.')
 private val versionSeparatorsPattern = versionSeparators.joinToString("", "[", "]")
 
-private val ignorablePrefixSuffixPattern = listOf("rel", "release", "final").joinToString("|", "(", ")")
+private val ignorablePrefixSuffix = listOf("rel", "release", "final")
```

Comment:
Could you help me understanding what the name means? e.g. why does it contain the word "prefix" ?

Reply:
Have a look at line 43, which makes it more clear: These string may appear as prefixes as well as suffixes to tags (like "rel4.12" or "4.12.rel") and be safely ignored when trying to find a version match. This change now allows to ignore these strings also as a suffix to the given version.